combat-trainer.lic

```diff
@@ -2606,6 +2606,9 @@ class AttackProcess
   def initialize(settings)
     echo('New AttackProcess') if $debug_mode_ct
 
+    @crossbow_is_repeater = nil # unknown, will detect when train a crossbow
+    echo("  @crossbow_is_repeater: #{@crossbow_is_repeater}") if $debug_mode_ct
```

Comment:
This new class property is not read from yaml config, not a whole lot of importance in logging that's it nil on init but the script was echoing the values of all the other properties so followed suit

Reply:
The other properties are only logged because they come from the YAML file. No need to echo nil.

Reply:
See line 2652 for an example of not echoing.